ddp-workspace/projects/ddp-dsm-ui/src/app/file-download/file-download.component.html

```diff
@@ -0,0 +1,38 @@
+<ng-container>
+  <ng-container *ngIf="downloading">
+    <div align="center">
```

Comment:
`align` attribute is obsolete (see [link](https://developer.mozilla.org/en-US/docs/web/html/element/div#attr-align:~:text=Note%3A%20The%20align%20attribute%20is%20obsolete%3B%20do%20not%20use%20it%20anymore.%20Instead%2C%20you%20should%20use%20CSS%20properties%20or%20techniques%20such%20as%20CSS%20Grid%20or%20CSS%20Flexbox%20to%20align%20and%20position%20%3Cdiv%3E%20elements%20on%20the%20page.)  ).
Use css instead (text-align: center) .